File: AgqrPlayer4Tv/AgqrPlayer4Tv/Components/Presenters/AgqrDetailsDescriptionPresenter.cs
using System;
using System.Reactive.Disposables;
using AgqrPlayer4Tv.Components.Fragments;
using Android.Support.V17.Leanback.Widget;
using Reactive.Bindings.Extensions;

namespace AgqrPlayer4Tv.Components.Presenters
{
    class AgqrDetailsDescriptionPresenter : AbstractDetailsDescriptionPresenter
    {
        private CompositeDisposable _disposables = new CompositeDisposable();

        protected override void OnBindDescription(ViewHolder viewHolder, Java.Lang.Object objectValue)
        {
            var nowPlaying = ((PlaybackControlsRowViewModel)objectValue).NowPlaying;

            viewHolder.Title.Text = " ";
            viewHolder.Subtitle.Text = " ";
            viewHolder.Body.Text = " ";

            nowPlaying.Title.ObserveOnUIDispatcher().Subscribe(x => viewHolder.Title.Text = x).AddTo(this._disposables);
            nowPlaying.Subtitle.ObserveOnUIDispatcher().Subscribe(x => viewHolder.Subtitle.Text = x).AddTo(this._disposables);
            nowPlaying.Body.ObserveOnUIDispatcher().Subscribe(x => viewHolder.Body.Text = x).AddTo(this._disposables);
        }

        public override void OnUnbindViewHolder(Presenter.ViewHolder viewHolder)
        {
            this._disposables.Dispose();
            base.OnUnbindViewHolder(viewHolder);
        }
    }
}

File: AgqrPlayer4Tv/app/src/main/java/org/misuzilla/agqrplayer4tv/component/fragment/presenter/AgqrDetailsDescriptionPresenter.kt
package org.misuzilla.agqrplayer4tv.component.fragment.presenter

import androidx.leanback.widget.AbstractDetailsDescriptionPresenter
import androidx.lifecycle.LifecycleOwner
import org.misuzilla.agqrplayer4tv.component.fragment.PlaybackControlsRowViewModel

class AgqrDetailsDescriptionPresenter(private val lifecycleOwner: LifecycleOwner) : AbstractDetailsDescriptionPresenter() {
    override fun onBindDescription(viewHolder: ViewHolder, item: Any?) {
        val nowPlaying = (item as PlaybackControlsRowViewModel).nowPlaying

        viewHolder.apply {
            title.text = ""
            subtitle.text = ""
            body.text = ""
        }

        nowPlaying.apply {
            getTitle().observe(lifecycleOwner, { viewHolder.title.text = it })
            getSubTitle().observe(lifecycleOwner, { viewHolder.subtitle.text = it })
            getBody().observe(lifecycleOwner, { viewHolder.body.text = it })
        }
    }
}